atlasdb-cli/src/test/java/com/palantir/atlasdb/cli/output/OutputPrinterTest.java

```diff
@@ -29,27 +29,27 @@
     public void testInfoPrintingWorksWithSingleReplacement() {
         String systemOut = StandardStreamUtilities.wrapSystemOut(
                 () -> print.info("Test this gets {}", "replaced"));
-        assertThat(systemOut).isEqualTo("Test this gets replaced ");
```

Comment:
These print an extra space at the end on windows; I have no clue why; best guess would be that something handles \r\n weird and it ends up printing out one of them as a space later?